src/r0/message/get_message_events.rs

```diff
@@ -47,8 +47,8 @@ ruma_api! {
         #[ruma_api(query)]
         pub limit: Option<UInt>,
         /// A RoomEventFilter to filter returned events with.
-        #[serde(skip_serializing_if = "Option::is_none")]
         #[ruma_api(query)]
+        #[serde(with = "crate::serde::json_string", skip_serializing_if = "Option::is_none")]
```

Comment:
Since we recently discovered that missing `Option`s with `serde(with)` fail deserialization.

```suggestion
        #[serde(
            with = "crate::serde::json_string",
            default,
            skip_serializing_if = "Option::is_none"
        )]
```